doc/nvim-tree-lua.txt

```diff
@@ -1425,10 +1653,10 @@ Navigation for marks is not bound by default in nvim-tree because we don't
 want to focus the tree view each time we wish to switch to another mark.
 
 This requires binding bookmark navigation yourself.
-
--- in your lua configuration
-vim.keymap.set("n", "<leader>mn", require("nvim-tree.api").marks.navigate.next)
-vim.keymap.set("n", "<leader>mp", require("nvim-tree.api").marks.navigate.prev)
-vim.keymap.set("n", "<leader>ms", require("nvim-tree.api").marks.navigate.select)
-
- vim:tw=78:ts=4:sw=4:et:ft=help:norl:
+>
+ vim.keymap.set("n", "<leader>mn", require("nvim-tree.api").marks.navigate.next)
+ vim.keymap.set("n", "<leader>mp", require("nvim-tree.api").marks.navigate.prev)
+ vim.keymap.set("n", "<leader>ms", require("nvim-tree.api").marks.navigate.select)
+<
+------------------------------------------------------------------------------
+ vim:ft=help:et:tw=78:ts=4:sw=4:norl:
```

Comment:
This appears unchanged; is there anything we can add to assist with standardisation?

Reply:
I don't get, what you mean. I place `ft=help` first because I found out that all options before `ft` don't recognize, and their default values are used.

Reply:
> To create the script to make such formatting with sed is a lot of work that not worth it. And the result of the script will never be like manually tailored. I have already done most of the work, so to maintain the existing docs won't be much work, since I doubt, that you will frequently change all options names and descriptions.

The script is already present. It updates `nvim-tree-setup` and `nvim-tree-default-mappings`. Fixing this script is not optional.

Reply:
> I found out that all options before `ft` don't recognize, and their default values are used.

Thank you.

RE additional options: any other formatting options such as `copyindent`